puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-katello-test.yml

```
@@ -1,45 +1,80 @@
+---
 - job-template:
-    name: 'foreman-katello-{version}-test'
-    project-type: pipeline
+    name: foreman-katello-{version}-{distro}-{action}-test
+    project-type: 'pipeline'
     sandbox: true
+    parameters:
+      - string:
+          name: expected_version
+          default: 'nightly'
```

For it to pass, I think this should match https://github.com/theforeman/foreman/blob/develop/VERSION (and foreman-installer and foreman-proxy which have the same content):
```suggestion
          default: '2.1-develop'
```